sys/ztimer/auto_init.c

```diff
@@ -52,65 +53,150 @@
 #define FREQ_1MHZ       1000000LU
 #define FREQ_250KHZ     250000LU
 #define FREQ_1KHZ       1000LU
+#define FREQ_1HZ        1LU
+
+#if CONFIG_ZTIMER_USEC_TYPE_PERIPH_TIMER
+//#if MODULE_ZTIMER_PERIPH_TIMER
```

Comment:
```suggestion
```

Reply:
I will change these when the first Murdock report is ready